src/main/groovy/edu/oregonstate/mist/catalogapitest/CatalogAPITestApplication.groovy

```diff
@@ -10,23 +11,45 @@ import io.dropwizard.setup.Bootstrap
 import io.dropwizard.setup.Environment
 import org.skife.jdbi.v2.DBI
 
+/**
+ * Main application class.
+ */
 class CatalogAPITestApplication extends Application<CatalogAPITestApplicationConfiguration>{
 
+    /**
+     * Initializes the application bootstrap.
+     *
+     * @param bootstrap
+     */
     @Override
     public void initialize(Bootstrap<Configuration> bootstrap) {
```

Comment:
the signature of the overridden method is `void initialize(Bootstrap<CatalogAPITestApplicationConfiguration> bootstrap)`